<!DOCTYPE html>
<!-- Designed by Mika Sarkin Jain. mikasarkinjain@gmail.com -->

<html lang="en" style="" class=" js no-touch csstransforms csstransitions">
<head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Applied Learning Initiative</title>
<link rel="icon" type="image/png" href="./assets/icon.png">

<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="ALI connect Stanford students to high‑impact projects">
<meta name="author" content="Stanford University">

<link rel="stylesheet" href="./assets/bootstrap.min.css" type="text/css">
<link rel="stylesheet" href="./assets/font-awesome/css/font-awesome.min.css" type="text/css">
<link rel="stylesheet" href="./assets/base.min.css" type="text/css">
<link rel="stylesheet" href="./assets/custom.css" type="text/css">
<!--[if lt IE 9]>
  <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
<!--[if IE 8]>
  <link rel="stylesheet" type="text/css" href="assets/cardinal/css/ie/ie8.css" />
<![endif]-->
<!--[if IE 7]>
  <link rel="stylesheet" type="text/css" href="assets/cardinal/css/ie/ie7.css" />
<![endif]-->
<script src="./assets/jquery.min.js"></script> 
<script src="./assets/modernizr.custom.17475.js"></script> 
<script src="./assets/bootstrap.min.js"></script> 

<!--[if lt IE 9]>
	<script src="assets/cardinal/js/respond.js"></script>
<![endif]--> 

<script src="./assets/base.js"></script>
<script src="./assets/custom.js"></script>
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-53864876-3"></script>
<script>window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'UA-53864876-3');</script>
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-53864876-4"></script>
<script>window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'UA-53864876-4'); </script>
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-53864876-5"></script>
<script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'UA-53864876-5'); </script>
	</head>

<body class="home">
<div id="su-wrap">
  <div id="su-content" style="padding-bottom: 320px;">
    
    <div id="top"></div>
    
    <div id="header" class="clearfix" role="banner">
      <div class="container">
        <div class="row">
          <div class="col-md-8"> 
            <div id="logo" class="clearfix"> <a href=".">
            	<img src="assets/logo.svg">
            </a> </div>
            <div id="signature">
              <div id="site-name">
                <a href=".">
                  <span id="site-name-1">Applied Learning Initiative</span>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    
    <div id="mainmenu" class="clearfix" role="navigation">
      <div class="container">
        <div class="navbar navbar-default"> 
          
          <button type="button" class="navbar-toggle btn-navbar" data-toggle="collapse" data-target=".navbar-collapse"> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="menu-text">Menu</span></button>
          
          <div class="navbar-collapse collapse" aria-expanded="false"> 
            <div role="navigation">
              <div id="primary-nav">
                <ul class="nav navbar-nav" aria-label="primary navigation">
                  <li id="nav-1"><a href=".">Overview</a></li>
                  <li id="nav-home"> <a href="./projects">Open Projects</a> </li>
                  <li id="nav-2"> <a href="./selfdesigned">Self-Designed <sup><font size="2" color="#f5541e">NEW</font></sup></a> </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        
      </div>
    </div>
    
    <div id="content" class="container" role="main" tabindex="0">
      <p>&zwj;</p><h1 class=""><center>Open Projects 2020&ndash;2021</center></h1>
      <div class="row"> 
        <div id="main-content" class="col-md-7 col-md-push-3b" role="main" left="25%">
		
	  <p>&zwj;</p><p><h2>Cloud Based Monitoring of Infant Language Environment</h2>
          Division of Neonatal and Developmental Medicine, Department of Pediatrics, Stanford
          </p>
          <p>
          Measuring the language environment of children, particularly infants, is important to understand its impact on brain development and future neurodevelopmental outcomes. Infants born prematurely, as much as 3 months early, are particularly at risk for problems with language and brain development. We work with a device, Starling, that measures the language environment of the neonatal intensive care unit, relating those measures to health outcomes. The Starling is a small wearable device that uses speech recognition technology to automatically provide continuous counts of the surrounding language environment across multiple days. The device connects with a smartphone app via bluetooth and then delivers data to an AWS server. The creators of the device have gone out of business but have gifted us with devices and access to the server. We are working with other researchers who are working on advancing the device functionality. We are looking for help in maintaining device functionality and coordinating with the central database server, dealing with system upgrades, etc. We are also looking for help with large scale data analytics and processing of our speech, clinical (eg., cardiorespiratory) and neuroimaging (diffusion and quantitative T1 MRI) data.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/cN5uM61Tm7ABVPcj8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>
         

	  <p>&zwj;</p><p><h2>Non-invasive BP sensing</h2>
          Division of Neonatal and Developmental Medicine, Department of Pediatrics, Stanford
          </p>
          <p>
          In close partnership with industry, we have developed and tested a small wearable, watch-like device that can capture blood pressure from extremely small babies who sometimes weigh ~500 grams (< 1.5 cans of coke). Data from the sensor, called pulse waveform data is information rich. Through a series of steps, it is compared with arterial waveform data. Our vision is that some day you can avoid the invasive procedures and just use the non-invasive sensor. Our preliminary data are very encouraging. Opportunities to analyze the arterial waveform characteristics are available, perhaps in isolation or in conjunction with other databases such as physionet.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/7DSwstt5tcKqEyia6"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>
         
	  <p>&zwj;</p><p><h2>Regional Tissue Oxygenation in the Neonatal Intensive Care Unit</h2>
          Division of Neonatal and Developmental Medicine, Department of Pediatrics, Stanford
          </p>
          <p>
          Near-infrared spectroscopy (NIRS) is used as a bedside technology in the neonatal intensive care unit (NICU) for non-invasive monitoring of regional tissue oxygenation in vulnerable babies.  As such, we have routinely monitored brain and kidney oxygen levels for the first week of life in over 500 infants over the past 5+ years.  These NIRS datafiles are a rich repository for research questions to help premature and critically ill babies in the NICU.  Several projects are available involving development of an algorithm to concatenate, process, and summarize NIRS data with an emphasis on data integrity.  Development of a graphical user interface for future work is a potential goal.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/7DSwstt5tcKqEyia6"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Prostate Cancer Grading on Histopathology Images</h2>
          Department of Radiology, Stanford
          </p>
          <p>
          Although radiology images, such as MRI, are very useful to identify suspicious lesions, tissue samples from biopsy or surgery, remain the only way to diagnose cancer. A pathologist visually inspects the histopathology images of the tissue sample and assesses the presence of cancer, thereby diagnosing the disease, and categorizes its aggressiveness by providing a grade. There is a need to automate the diagnosis and grading of prostate cancer on histopathology images to limit the subject interpretation of images and allow for high throughput annotation of histopathology images. This project will use public data release as part of a MICCAI challenge and will focus on developing deep learning methods to distinguish different grades of prostate cancer based on histopathology images. The dataset includes 244 images with Gleason grades regional labels provided by 6 pathologists. Data and additional information: https://gleason2019.grand-challenge.org/
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/ARBrqjhZKeJeKawB7"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Prostate Cancer Detection on MRI</h2>
          Department of Radiology, Stanford
          </p>
          <p>
          Despite technical advancements in the diagnosis and treatment of prostate cancer, it remains the most common and the second most deadly cancer in men in US [1]. MRI plays an essential role in the diagnosis of prostate cancer, yet the radiologists’ interpretation of the MRI images has a relatively high sensitivity, 0.85-0.95, yet a reduced specificity, 0.71-0.75, to detect prostate cancer. There is a need to develop advanced machine learning methods to accurately detect cancer on MRI. This project will use public data released as part of the prostateX challenge and will focus on developing deep learning methods to distinguish benign conditions (non-cancer) from cancer on multi-parametric MRI. The training dataset includes 204 patients with 331 findings categorized as cancer or benign. Data and additional information: https://wiki.cancerimagingarchive.net/display/Public/SPIE-AAPM-NCI+PROSTATEx+Challenges#28ab80368ed440fb9a26afa8bbef487a
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/tCjJwUrDQZCDZuJF9"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Development of Brain-Based Biomarkers for Pain</h2>
          Division of Pain Medicine, Department of Anesthesiology, Perioperative and Pain Medicine, Stanford
          </p>
          <p>
          There is an urgent need for the development of biomarkers of chronic pain to better predict (1) the best treatment response for an individual patient; and (2) whether a person’s pain will improve or worsen over time. We have been developing neuroimaging-based biomarkers of chronic pain and amassed a large dataset of brain and spinal cord neuroimaging scans of people with chronic pain and matched healthy controls. There are multiple projects involving the use of traditional and machine learning approaches for the development of prediction models. We ultimately wish to translate this knowledge into clinical decision support tools to realize the vision of precision pain medicine where can provide the best treatment for a particular patient under a specific circumstance. For those interested in applying to medical school or a health related PhD program, we have a long experience with providing shadowing at the Stanford Pain Management Center. We can help with applications and have a long track record in helping students get accepted into these programs. Furthermore, plenty of opportunities for students to either lead or participate in projects leading to a manuscript publication. For more information on our research group please go to: http://snapl.stanford.edu
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/MTPiZ3XnQQ1yw5Qi8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Optimizing Patient Care and Innovative Research Using an Open Source Learning Healthcare System</h2>
          Division of Pain Medicine, Department of Anesthesiology, Perioperative and Pain Medicine, Stanford
          </p>
          <p>
          Chronic pain is an astounding problem affecting 100 million Americans and costing our country half a trillion dollars per year. One of the major challenges in understanding and treating pain is the need for better quality data. To address this problem, the National Academy of Medicine (NAM) called for the development of learning healthcare systems (LHSs) which bring together clinicians, scientists, and patients work together to capture high-quality data. In essence, to turn our clinics into research labs where every patient encounter is deeply characterized. To answer this call from the NAM, we developed CHOIR (http://choir.stanford.edu) as an open-source (free) LHS. Since its introduction in 2012, we have collected many 10’s of thousands of baseline and longitudinal data on real-world patients with pain (and other health conditions). We have also implemented CHOIR in multiple academic institutions in the US, Canada and Israel. Our ultimate goal is to realize the vision of precision pain medicine where can provide the best treatment for the particular patient under a specific circumstance. Furthermore, all development efforts serve to benefit the other clinical specialties also using CHOIR. Multiple projects including machine learning, data visualization and platform development are provided in the apply link.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/12BK8b1FKg5pUTdV9"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Mignot Lab: Proteomics in the CSF and Blood in Relation to Sleep Disorders</h2>
          Center for Narcolepsy, Stanford Medicine
          </p>
          <p>
          The Mignot lab studies sleep and sleep disorders and we are analyzing large amounts of proteomic data (1,100 to 5,500 proteins) in the blood or CSF of patients with various pathology, together with genome wide association and exome sequencing data.   Goal is to link genetic to protein to disease.  This project requires knowledge of genetic analysis and associated statistics. Applicants who lack specialized skills in genetic analysis and statistics may still apply if they indicate willingness to commit for multiple quarters. This will allow time for mentorship and a significant contribution to the lab. Python programing needed.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/3TrTGiAd7ozxkatK8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Mignot lab: Immunogenetics and Immunology of Narcolepsy</h2>
          Center for Narcolepsy, Stanford Medicine
          </p>
          <p>
          Narcolepsy is an autoimmune disease, and we conducted genome wide association in over 5000 cases and exome sequencing data in many familial cases.  The disease appears to be a T cell mediated attack on hypocretin neuron elicited by a cross-reactivity with a flu antigen.  Project involves data analysis of work with T cell, Fluorescence cell sorting (FACS), sequence analysis. This project requires either (1) hands on wet lab immunology or (2) knowledge of genetic and sequence analysis and associated statistics with python programing needed. Applicants who lack specialized skills in wet lab immunology or genetic analysis may still apply if they indicate willingness to commit for multiple quarters. This will allow time for mentorship and a significant contribution to the lab.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/3TrTGiAd7ozxkatK8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Mignot Lab: Genetics and Pathophysiology of NMDA Encephalitis and Brain Paraneoplastic Syndromes</h2>
          Center for Narcolepsy, Stanford Medicine
          </p>
          <p>
          NMDA encephalitis is an autoimmune disease characterized by psychosis and seizure. It can occur in the context of ovarian teratomas, although more often it is not associated with any tumor.   Other paraneoplastic syndrome we study are ataxias or other brain encephalitis.  We conducted genetic analysis on large number of cases and have findings indicating specific immunological mechanism that need to be followed up through additional genetic analysis such as HLA imputation or immunological studies. Applicants who lack specialized skills in genetic analysis and statistics may still apply if they indicate willingness to commit for multiple quarters. This will allow time for mentorship and a significant contribution to the lab. Python programing needed.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/3TrTGiAd7ozxkatK8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>
          

          <p>&zwj;</p><p><h2>Mignot Lab: Machine Learning for Automated Scoring of Polysomnography Studies</h2>
          Center for Narcolepsy, Stanford Medicine
          </p>
          <p>
          We have gathered thousands of polysomnography studies are conducting machine learning studiesto score recordings automatically (supervised machine learning) , linking this data with genetics (genome wide association) or phenotype (various disease and sleep disorder). We have a team of 3-5 electricalengineers working on this and numerous projects in this space.   Requirement is ability to finish a project that is not just a demonstration project but creates something that can be built upon and includes a publication.   Python programing needed.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/3TrTGiAd7ozxkatK8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Gene Expression Data Organization and Mining</h2>
          Human Immune Monitoring Center, Institutefor Immunity, Transplantation, and Infection
          </p>
          <p>
          The Human Immune Monitoring Center (HIMC)has generated gene expression data from blood cellsfor hundreds of healthy individuals across a range of ages, in studies profiling healthy immunity and response to influenza vaccination. Our online database, Stanford Data Miner (SDM), was recently updated to allow upload and mining of such data. This projectwould involve: (1) formatting existing data using an R script so that it is compatible with SDM and contains the relevant subject IDs; (2) upload of the data and mapping to subject IDs in SDM; (3) quality control and visualization of the uploaded data using novel tools in SDM; and (4) potentially defining new metrics of healthy blood gene expression from the uploaded data. Experience with running R scripts is needed, and experience with databases in general wouldbe helpful. Academic credit can be arranged, and a commitment for 2 quarters is preferred.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/wFEbSFCaLSgsQzBW6"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Machine Learning to Predict Cardiovascular Disease</h2>
          Stanford University School of Medicine
          </p>
          <p>
          Corneal Arcus (CA - a white ring around the edge of the iris) has been correlated with high cholesterol particularly in younger people but not yet with coronary calcium scores. In our project, we will be recruiting subjects who have had their coronary calcium CT scans already done to capture photos of their irises using their phones.  We will then correlate the presence and severity of CA with cholesterol, coronary calcium and diabetes.  We will then train a deep learning model to predict cholesterol andcoronary calcium from iris photos captured on smartphones.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/1Q4TW6soNQWUXQ2N8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>IMPACT Study</h2>
          Stanford Center for Asian Health Researchand Education (CARE)
          </p>
          <p>
          The Initiate and Maintain Physical Activity in Clinics (IMPACT) study is a 2.5-year study designed to examine the efficacy of structured exerciseprograms offered in the clinic setting to help patients manage their type 2 diabetes and meet exercise recommendations. Patients are assigned into one of three arms: weekly exercise 1x, 3x, or 0x (usual care). The goal is to determine which exercise regimen will most help patients adhere to exercise and lower their HbA1c values. This lab offers opportunities toconduct patient clinical visits, author papers and grants, and more.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/tFTctAR6tKcvdQup8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>STRONG-D Study</h2>
          Stanford Center for Asian Health Researchand Education (CARE)
          </p>
          <p>
          Normal weight individuals with type 2 diabetes (NWD) have higher risk of mortality than overweight or obese individuals. NWD may have less lean muscle mass in proportion to adipose tissue, which may contribute to this higher risk of mortality. Strength training increases muscle mass rather than reducing body weight and can combat the age-related shiftin body composition that results in higher fat massand decreased muscle mass (i.e., sarcopenia). Thus,strength training may therefore be helpful for NWD rather than weight loss, as is recommended for overweight/obese individuals with type 2 diabetes (T2DM).The goal of the Strength Training Regimen fOr Normal weiGht Diabetics (STRONG-D) Study is to determine the best exercise regimen for NWD, who experience greater mortality than overweight/obese diabetics. Patients are randomized into one of three arms: Strength, Aerobic, or Combo. This lab offers opportunities to conduct patient clinical visits, author papers and grants, and more.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/JJimvxKhR8CwZMEi8"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Functional Ca2+ Image Analysis based on In Vivo Cochlear Hair Cell Imaging</h2>
          Stanford, Department of Otolaryngology Head & Neck Surgery
          </p>
          <p>
          In vivo imaging with cellular resolution enables us to identify and characterize receptive field properties of a sensory system. However, cochlear in vivo imaging remain challenging, owing to not only its deep location but also bony structure filledwith fluids. The goal of our project is to develop in vivo cochlea imaging to characterize how sound istransferred across sensory hair cells. So far, we’ve developed a method to resolve individual cochlear hair cells with hearing function in live mice, and are collecting functional data. We are looking for a keen research assistant at any level that is eager to learn, well organized, and that can help out with research studies on an ongoing basis. Coding skill would be preferred. This position can be for undergraduate credit or a paid position.
          </p>
          <p><a class="more-link" target="_blank" href="https://forms.gle/Ess6bvj2XfKE2M7a7"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Machine Vision for theNeural Basis of Social Interactions</h2>
          Stanford Department of Neurobiology
          </p>
          <p>
          Understanding the neural basis of social interactions remains one of the great mysteries of modern neurobiology with profound implications for basic science and neurological and psychiatric applications. This project seeks to parse videos of interacting animals into a set of behavioral building blocks tied to both individual animal actions and group dynamics. Our goal is to combine multiple deep learning approaches including convolutional neural networks and recurrent neural networks or attention-based sequence models. As a second stage the dynamics of behavior revealed by the developed algorithms will be related to neural recordings and video processing will be adapted for real-time analysis. This project requires a two quarter minimum commitment.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfyLCgNpmtP1URtFOa5jCVtHBWQ1n1BG5QxoomP4JxciWC0qA/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Single Cell Transcriptomics of Tumor and Immune Biology</h2>
          Stanford Pathology Department and VA Medical Center
          </p>
          <p>
          Single cell RNAseq is an emerging technology that allows us to examine expression of all genes by thousands of cells, yielding unparalleled insights into cell biology, communication and responses.The Butcher lab is using scRNAseq to understand vascular and immune system responses and interactions in cancer and inflammation.  Student(s) will help improve existing tools for analysis and visualization of the massive data sets generated, and will help develop and apply novel algorithms for probing cell-cell interactions, gene and communication networks.   Projects will involve coding, data analysis and biological knowledge application and discovery.  Opportunities for publication and credit.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfNI49G84J1g9VAro-qkeqBkgdfW1zvKTAoYk_ov2nplPPaRQ/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Early Detection of Cerebral Ischemia</h2>
          Stanford Center for Biodesign
          </p>
          <p>
          Our project focuses on enabling the earlydetection of ischemic stroke in high-risk patients to increase access to life-saving treatments and reduce the disability and mortality associated with these often devastating events. We are looking for 2 talented students to assist us in the process of implementing state of the art algorithms for the processing of EEG signals and to contribute in the development of new methods and models for detecting cerebral ischemia. Must have: Passion to address unmet clinical needs, Fluency in Matlab and/or Python.  Nice to have: Prior experience with the Matlab Fieldtrip & EEGLAB toolkits and/or the Python MNE-Python. For more information, or to express interest, reach out to orvar@stanford.edu or ayo.roberts@stanford.edu.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfoHYkj_FPtk5jM77SBARkGy9fyGOCdz0c0SXpXrfg-81GnJA/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Opioid Safety: Personalizing Prescribing to Reduce Misuse</h2>
          Stanford University, Center for Population Health Sciences; Biomedical Data Science
          </p>
          <p>
          How many opioid pills should a doc prescribe for a patient in pain after back surgery or kneereplacement?  Does it vary by gender or age?  Opioids have run rampant in the US with some US states reporting more opioid prescriptions each year than residents.  In 2016, over 64,000 Americans died from overdoses, 21 percent more than the almost 53,000 in 2015.  If not remediated, public health experts estimate that nationwide over 500,000 people could die from the epidemic over the next 10 years.
          <br>
          How can we ensure the safe and effective prescribing of one of the most effective medicines doctors have?  We seek to inform the current discussion on opioid prescribing guidelines by using a range of analytic techniques to explore the relationship between pre-surgical patient characteristics, opioid use profiles and adverse opioid outcomes.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLScOL3eCOz27LDZs02QeVzuVP5XqtUOczDRMWCXHAOWvF553dQ/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Multi-Morbidity Clusters Associated with Mortality</h2>
          Stanford University, Center for Population Health Sciences; Biomedical Data Science
          </p>
          <p>
          Danish population healthcare data is considered the "test tube" for observational health analyses.  Using a nationally representative sample of Danish residents, this work seeks to use unsupervisedlearning methods and data visualization techniques to examine multi-morbidity complexes that are strongly associated with one and five year mortality.  Such insights can be used in a wide range of applications from drug design to policy making.  The data has already been cleaned, processed and is ready for analysis in feature matrices.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdE1mDKA4fsSGQHIKyU7aKa9A42KIFX4eElGZBvM7b2vNpQ_Q/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Computer Vision for Post-Surgical Wound Surveillance</h2>
          Stanford Medical School, Vascular Surgery
          </p>
          <p>
          Post-operative wound infections occur between 2% and 15%, depending on the surgery. This is considered a “never-event” and CMS does not reimbursethe treatment of such complications- similar to re-admissions which occur within 30-days of a procedure. There is great benefit to identifying signs of infection early for early intervention before the typical 1-week follow up. We have an IRB approved study to collect photos from wounds from post-op day 0 through post-op day 7 which we label and have the ability to run computer vision algorithms on. In addition to early wound infection detection we are very very interested in the art of taking a photo and converting that to a “structured report” that we would use for documentation in the electronic medical record asthis is what is critical for billing. In addition we have the opportunity to gain access to daily photos of patients in the ICU as well as in their rooms to automate the process of generating structured reports for documentation. An example would be, “Patientis asleep, intubated with naso-gastric tube in place.”<br>Two quarter minimum engagement (Summer internship an option)
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdfWl9OC-Em7Eo5ljhYdc12ueoLhsaYk5RVtEZ_jbEXxkLc9Q/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Automated Carotid Ultrasound Screening Using Deep Learning</h2>
          Stanford Medical School, Radiology
          </p>
          <p>
          Carotid ultrasound is a high volume screening tool to evaluate patients for stroke risk scoring. We have labeled datasets (performed by the interpreting radiologist) for normal/abnormal cases and detailed structured labels from the report dictation.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSeRQFSZLJVfrOMqBsTkyG11hTUkiCsibthDWE7Eqyr1L3hNrQ/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Clinical Validation ofa Passive Mobile Activity Tracking Algorithms to Assess Functional Capacity in Cardiovascular Disease Patients</h2>
          Stanford Medical School, Vascular Surgery
          </p>
          <p>
          We are several years into developing a mobile research app to study and clinically validate activity tracking using a patient’s personal mobile device +/- a smart watch. In addition to getting passive daily activity data (max steps without stopping,total daily steps, flights climbed) we also have high frequency (100Hz) raw accelerometer data and heart rate data for 6-minute walk tests in well characterized patients pre- and post- interventions. We are looking for 1) response curves in activity patterns to interventions, 2) characteristics in passive metrics relative to clinical state, and 3) correlation of passive activity to 6-minute-walk-test results among other things. We have data not only form our own studies here but also from a collaborative study with a large pharmaceutical company.<br>Two quarter minimum engagement (Summer internship an option)
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdnheQ06ql_izFU8nv_dZbFF1mkyZ5kMJh56OksoMvIh-Mq4g/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Therapeutic Radiation Technology</h2>
          Zap Surgical Systems, Inc.
          </p>
          <p>
          Zap Surgical Systems, Inc. is a San Carlos, CA-based developer and manufacturer of a new generation of stereotactic radiosurgery systems. Zap Surgical Systems set out to create a first-of-its-kind therapeutic radiation technology that would be “self-shielded” and not require a radiation therapy vaultunder most circumstances. Another critical company goal was to invent a device that met or exceeded thebest quality standards of existing radiosurgical instruments, i.e. there would be no compromise in terms of performance. It remains the goal of Zap Surgical Systems to provide the above revolutionary technology at the lowest possible price; the general benchmark the company has set for itself is that the cost of ownership worldwide should be approximately one-third the cost of competing radiosurgical modalities.Zap is engaged in active research on new uses for stereotactic radiosurgery. One of these is “radiomodulation,” precise, noninvasive modulation of dysfunctional brain circuits.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSf2MhZ6m7AsAQzrWiDIgJPT2kpSQuxp5eu4ae-GT5wigbciKg/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Selective Plane Illumination Microscopy (SPIM)</h2>
          Stanford Medical School, Cardiovascular Medicine
          </p>
          <p>
          Student(s) will assist in completing microscope hardware construction, including optomechanical integration, laser beam alignment, and computer/software integration, following the openspim design (http://openspim.org/Welcome_to_the_OpenSPIM_Wiki). After hardware construction, student(s) will create apipeline for storing/retrieving images, videos, andassociated data on Amazon Web Services (AWS) and for image processing/analysis of images, videos, and data. The data set sizes will range from a few GB upwards to TBs. Images and videos will be primarily from human pluripotent stem cell-derived tissues and organoids, along with rodent hearts and blood vessels.Experience with both hardware and software is preferred. A time commitment of at least two quarters (research credit can be arranged) is also preferred.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSexQ9vPc4NhHMBokl5kyi5NyR0tL3hDWDy34kQNj8V8Dvs2Mg/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Optical Mapping Microscopy for Arrhythmia Research</h2>
          Stanford Medical School, Cardiovascular Medicine
          </p>
          <p>
          Student(s) will assist in completing microscope hardware construction, including integration of a high speed camera, high power LEDs, a video projector for spatial optical illumination, optics, anda motorized XY stage. After hardware construction, student(s) will create a pipeline for storing/retrieving images, videos, and associated data on Amazon Web Services (AWS) and for image processing/analysis of images, videos, and data. Students will create data acquisition software/scripts (in LabView and/or Matlab) that integrate signals from the above devicesand instruments (open loop and closed loop systems). Images and videos will be primarily from human pluripotent stem cell-derived monolayer cardiovascular tissue as a model for investigating cardiac arrhythmias. Experience with both hardware and software is preferred. A time commitment of at least two quarters(research credit can be arranged) is also preferred.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfn6SIPLKh1X0xcDnMJl7NDNLJCmHVkvJbzJUP61ZeO_WEfIg/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Segmentation of Nucleiin Histological Sections</h2>
          Stanford University, Department of Bioengineering
          </p>
          <p>
          Automatic segmentation of nuclei is a keystep in digital histology. While the problem has been extensively studied, there is no well-establishedopen-source library/network. A successful project might become an essential element of many scientific workflows.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLScGHibdoBLx2jxjnYvOGZKUvMC17ARnvyX_d6FovFQC8kW-kw/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Disease Treatment Proposal</h2>
          Stanford University, Department of Bioengineering
          </p>
          <p>
          Design of a system that asks for symptoms. Then, based on symptoms and preprocessed embeddings of diseases (e.g. at https://en.wikipedia.org/wiki/List_of_infectious_diseases) it either diagnoses a disease or if the confidence is low, it asks for more details ("is the rash dark?"). A proof-of-concept exploratory project with many ambitious opportunities depending on interests!
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdbyU0RRYqAhWNzYjPk9PemwR_7HbxI7Dqa3bOjMD0ocjSVBw/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Automatic Evaluation of Burns</h2>
          Stanford Medical School, Plastic & Reconstructive Surgery
          </p>
          <p>
          Care of patients with burns is highly specialized and yet initial evaluation and triage of burn patients typically occurs in centers without available burn expertise. We have assembled the largest curated dataset of burn images that have been segmented and classified by experts. During last year’s CS231n we mentored a student who developed a pixel-lever classifier to evaluate burn size and depth. Our future work includes at least two opportunities: (1) develop a classifier for normal unburned skin, whichwill result in a clinically useful tool that could have exciting implications for initial treatment of these critically ill patients and (2) evaluate the effect of skin tone on performance of our classifier,whether underrepresentation of certain skin tones introduces bias and impairs performance of the classifier for certain patients, and explore opportunitiesto enhance performance for underrepresented groups.While we cannot provide computer science expertise,we worked very closely with our student last year to provide medical subject matter expertise, which resulted in a successful project that currently is in preparation for submission to a medical journal.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfZIt5VVA4JE1k_-Iwt-5iv3r_iimTcwoJo19YZ6c80ToGIuw/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Receptive Field Mapping</h2>
          SLAC National Accelerator Laboratory
          </p>
          <p>
          Given a neural network trained on an image recognition task, map out the regions of input space to which each neuron responds. Our approach uses nonlinear programming to solve this as a mathematical optimization problem. Or replace “image recognition task” with the task of your choosing.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSemT7OfOqH6YTwdaai1PG_vfXkkkYu-zFj4x1yX-7cf6JnfcQ/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Neural Network QualityMetrics</h2>
          SLAC National Accelerator Laboratory
          </p>
          <p>
          Given a trained neural network classifier, what is the rate of false positive classification?Can a statistical regularizer reduce the rate of false positives? False negatives?
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfApifs4SdpKWPYP0NUT-_zEwlCU8YtsDSPRDIESgD_xonJYg/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Spectral Analysis of Neural Networks</h2>
          SLAC National Accelerator Laboratory
          </p>
          <p>
          Given a trained neural network, what can be learned from examining the eigenvectors and eigenvalues of the weight matrices, and the Lyapunov exponents of the network? Can these measures be related (possibly by using machine learning) to the network quality metric and the receptive field maps?
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfimcH9AEVb7V3AIR3qDz_84z9swuCakr0WUtQvEJAlOWbzDQ/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Neural Network Explanations</h2>
          SLAC National Accelerator Laboratory
          </p>
          <p>
          Given a trained neural network, generate an English language description of network decisions. The approach is to use machine learning based on the information from the preceding three projects.
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSe_8jHCFgXBJvKwr2LPhsEVGAanO92v3KYONzcSXBgMRTYuFg/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p><p><h2>Inferring Patient Outcomes from Healthcare Variables</h2>
          Stanford University, Department of Anesthesiology
          </p>
          <p>
          Use existing healthcare insurance claims data to investigate the association between healthcare variables and patient outcomes (datasets contain 10s -100s of millions of patients).
          </p>
          <p><a class="more-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdynI_mCDoXKJZ3qTkEAtKlPzRU4-1zzFtgELM7bHE_QN35gQ/viewform"><i class="fa fa-chevron-right"></i> <span>Apply</span></a></p>


          <p>&zwj;</p>
          </div>
      </div>
    </div>
    
  </div>
</div>

<div id="footer" class="clearfix footer" role="contentinfo" style="margin-top: -320px;">
  <div class="container">
    <div id="footer-content" class="row">
      <div class="col-xs-12 col-sm-6 col-sm-push-3">
        <h3>Partners</h3>
        <p><img src="./assets/partners.png" width="97%"></p>
      </div>
      <div class="col-xs-12 col-sm-3 col-sm-push-3">
        <h3>People</h3>
        <table>
		  <tr>
		    <td width="42%" valign="top"><font color="#696969"><b>President</b></font></td>
		    <td>Mika Sarkin Jain</td> 
		  </tr>
		  <tr>
		  	<td valign="top"><font color="#696969"><b>Vice President</b></font></td>
		    <td>Tom Pritsky</td> 
		  </tr>
		  <tr>
		  	<td valign="top"><font color="#696969"><b>Outreach</b></font></td>
		    <td>Neha Srivathsa, Levi Lian, Matthew Prospero</td> 
		  </tr>
		  <tr>
		  	<td valign="top"><font color="#696969"><b>Faculty Advisors</b></font></td>
		    <td>Professor Tarik Massoud, Professor Oliver Aalami</td> 
		  </tr>
		</table>
      </div>
      <div class="col-xs-12 col-sm-3 col-sm-pull-9">
        <h3>Contact Us</h3>
        <p><br>If you have a question or suggestion, or want to work with us, we'd love to hear from you!</p>
        <p><a class="more-link" target="_blank" href="mailto:info@appliedlearninginitiative.org"><i class="fa fa-chevron-right"></i> <span>info@appliedlearninginitiative.org</span></a></p>
      </div>
    </div>
  </div>
    
</div>

<div class="size-detect-xs"></div><div class="size-detect-sm"></div><div class="size-detect-md"></div><div class="size-detect-lg"></div></body></html>
